annex/examples/annexA/src/example7.cpp

```diff
@@ -0,0 +1,50 @@
+/****************************************************************************
+*	1 event loop
+*	Listener to print the content of the event and the thread id
+*	Publish and subscriber for other types of data: int, my own strct, etc.
+*	Number as the output
+*****************************************************************************/
+
+#include <iostream>
+
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
+
+#include <thread>
+
+#include <chrono>
+
+# include "int_publisher.h"
+
+# include "position_publisher.h"
+
+int main() {
+	kpsr::high_performance::EventLoopMiddlewareProvider<16> eventloop(nullptr);
+
+	eventloop.start();
+	
+	std::thread::id this_id = std::this_thread::get_id();
+	std::cout << "t1's id: " << this_id << '\n';
```

Comment:
t1 doesn't exist now. It should be "main thread id". I think you can remove line 25 and line 26 both.

Reply:
Done